src/calculateRentalCost.js

```diff
@@ -5,6 +5,24 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const oneDayCost = 40;
+  const threeDays = 3;
+  const sevenDays = 7;
+  const minTermDiscount = 20;
+  const longTermDiscount = 50;
+  const totalPrise = days * oneDayCost;
+
+  if (days < threeDays) {
+    return totalPrise;
+  }
+
+  if (days >= threeDays && days < sevenDays) {
+    return totalPrise - minTermDiscount;
+  }
+
+  if (days >= sevenDays) {
```

Comment:
Consider using `else if` for the second condition since the conditions are mutually exclusive. This can slightly improve the efficiency and readability of the code.